src/coreclr/jit/codegenriscv64.cpp

```diff
@@ -3327,7 +3351,88 @@ void CodeGen::genCodeForCompare(GenTreeOp* tree)
     if (varTypeIsFloating(op1Type))
     {
         assert(tree->OperIs(GT_LT, GT_LE, GT_EQ, GT_NE, GT_GT, GT_GE));
-        NYI_RISCV64("genCodeForCompare-----unimplemented on RISCV64 yet----");
+        bool      IsUnordered = (tree->gtFlags & GTF_RELOP_NAN_UN) != 0;
```

Comment:
This is wrong naming conventions (should be `isUnordered`), but let's not hold this PR up on it.

Reply:
Thank you! I will fix in the next PR.